### DISCUSSION TITLE
Aqara Opple 4/6 button wall switch - multiple endpoints?

### DISCUSSION THREAD

**Discussion by SadGamerGeek** [Koenkk/zigbee2mqtt#12762] [Devices]
I was hoping I could use my Aqara Opple 6 button switch to bind to multiple lights. In the Bind menu for it, I see there are 6 endpoints, which I initially thought would related to the 6 buttons. Only endpoint 1 shows any clusters though. When I bind that to a light, all three button pairs are then used (on/off, dim up/down, colour temp up/down). Is there any meaning to those other (2-5) endpoints? Is there any way to configure it in a way to use them? If not, can anyone recommend any alternative wall switches/remotes that provide multiple endpoints so I can bind to more than one device independently? 

I realise I could do this with automations, but I'm trying to use binding for core functionality (switching lights on/off) for resilience.

**Comment by privatesam** [comment]
I have been using deconz/phoscon for years with these buttons.  Recently I wanted to migrate to zigbee2mqtt and discovered this massive limitation!  It was a showstopper so I went back to deconz.  There must be a way to bind the individual buttons to different lights if you can do it in deconz.
